src/control/cmd/dmg/pretty/pool_test.go

```diff
@@ -30,14 +30,15 @@ func TestPretty_PrintPoolQueryResp(t *testing.T) {
 		"empty response": {
 			pqr: &control.PoolQueryResp{},
 			expPrintStr: `
-Pool , ntarget=0, disabled=0, leader=0, version=0
+Pool , ntarget=0, disabled=0, leader=0, version=0, state=
```

Comment:
We should try to print something sane even if the state is empty. Maybe "unknown"?